featuretools_tsfresh_primitives/primitives/fft_coefficient.py

```diff
@@ -40,3 +45,123 @@ def function(x):
             return list(fft_coefficient(x, param=param))[0][1]
 
         return function
+
+
+class ShortTermFftCoefficient(TransformPrimitive):
+    """Calculates the the fourier coefficients of the one-dimensional discrete
+    Fourier Transform for real input by fast fourier transformation algorithm over a given window.
+
+    .. math::
+        A_k =  \\sum_{m=0}^{n-1} a_m \\exp \\left \\{ -2 \\pi i \\frac{m k}{n} \\right \\}, \\qquad k = 0,
+        \\ldots , n-1.
+
+    The resulting coefficients will be complex, this feature calculator can
+    return the real part (attr=="real"), the imaginary part (attr=="imag), the
+    absolute value (attr=""abs) and the angle in degrees (attr=="angle).
```

Comment:
The quotes on the attr examples here need to be cleaned up.

Reply:
Looks like they need to be changed in the original as well. Good catch